### DISCUSSION TITLE
Dev and Prod

### DISCUSSION THREAD

**Discussion by stefan-athleads** [supabase/supabase#1024] [Questions]
When starting a new project, how do you guys go about separating your dev and prod DBs? Do you create a new project per environment and then manage them client-side with environment variables?

For context, I am coming from a firebase background and wanting to switch over to supabase.

**Comment by silentworks** [comment]
If dev is your own isolated environment then you can use the newly release supabase cli and get a local docker environment setup for dev work. https://supabase.io/blog/2021/03/31/supabase-cli
